dev/integration_tests/deferred_components_test/run_release_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
any thoughts about using Dart for this, similar to some of the other integration tests? For example:
https://github.com/flutter/flutter/blob/master/dev/devicelab/lib/tasks/dart_plugin_registry_tests.dart#L156

Reply:
Only one of the commands is a flutter build command, everything else is a mix of adb, bundletool, etc. I think a shell script may be clearer and more concise for now at least.